woodwork/logical_types.py

```diff
@@ -448,6 +466,15 @@ def transform(self, series):
 
         return super().transform(series)
 
+    def validate(self, series, return_invalid_values=True):
+        super().validate(series, return_invalid_values)
+        if not _is_valid_latlong_series(series):
+            raise TypeValidationError(
```

Comment:
`return_invalid_values` is not being used at all. You can look at this example https://github.com/alteryx/woodwork/blob/6f323b6f37f44cf38b5ee18a65931b600f02e5e2/woodwork/logical_types.py#L694 to see how to return the invalid values, which is necessary for how `LogicalType.validate` gets used in `TableAccessor.validate_logical_types`.

Reply:
removed the parameter